official/transformer/v2/transformer_main.py

```diff
@@ -125,24 +125,38 @@ def train(self):
         map_data_fn, num_parallel_calls=params["num_parallel_calls"])
 
     init_epoch = flags_obj.init_epoch or 0
-    init_steps = init_epoch * flags_obj.steps_per_epoch
-    callbacks = self._create_callbacks(cur_log_dir, init_steps, params)
+    init_steps = init_epoch * flags_obj.train_steps
+    callbacks = self._create_callbacks(flags_obj.model_dir, init_steps, params)
 
     history = model.fit(
         train_ds,
-        initial_epoch=init_epoch,
```

Comment:
erm, i am not sure if the logic here is all consistent if init_epoch and init_steps are being used.. maybe would be good for @lintian06 to confirm

Reply:
I would like to see this stream lined to "just work" like the resnet example; we should not need to say what epoch we are starting at to start up training again.  It may take some iterations.

Reply:
I have fixed this and it does not allow starting from a checkpoint.  We can add that back and it should just work and not require you to know what epoch you were on.  The previous code only worked if you knew where you stopped.

Reply:
Hi Toby, the default init_epoch is 0 and just works as another other models, so no need to set or remove.